Plugins/NativeInput/Source/Shared/NativeInput.cpp

```diff
@@ -16,6 +16,17 @@ namespace Babylon::Plugins
         constexpr uint32_t POINTER_X_INPUT_INDEX{0};
         constexpr uint32_t POINTER_Y_INPUT_INDEX{1};
         constexpr uint32_t POINTER_BUTTON_BASE_INDEX{2};
+        constexpr uint32_t POINTER_BUTTON_LEFT_CLICK{2};
+        constexpr uint32_t POINTER_BUTTON_MIDDLE_CLICK{3};
+        constexpr uint32_t POINTER_BUTTON_RIGHT_CLICK{4};
+        constexpr uint32_t POINTER_BUTTON_BACK_CLICK{5};
+        constexpr uint32_t POINTER_BUTTON_FORWARD_CLICK{6};
+        constexpr uint32_t POINTER_MOUSEWHEEL_X{7};
+        constexpr uint32_t POINTER_MOUSEWHEEL_Y{8};
```

Comment:
Seems like none of these are used. Remove?

Reply:
They will be used but since they're not used yet, I can remove them for the being.  Removed.

Reply:
After testing, I needed to add them back because the JS side references the inputs by array index so we need the values up to POINTER_MOVE